extensions/ql-vscode/src/view/result-table-utils.tsx

```diff
@@ -142,7 +142,9 @@ export function nextSortDirection(direction: SortDirection | undefined, includeU
 }
 
 export function emptyQueryResultsMessage(): JSX.Element {
-  return <span>
-    This query returned no results. If this isn&apos;t what you&apos;re expecting, and for effective query-writing tips, check out the <a href="https://codeql.github.com/docs/codeql-language-guides/">CodeQL language guides</a>.
-  </span>;
+  return <div style={{ height: '300px', display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+    <span style={{ maxWidth: '80%', fontSize: '14px', textAlign: 'center' }}>
```

Comment:
Please avoid inline styles. Use classes instead. Add the styles to `resultsView.css` as a class and use the classes here.